src/test/java/FlashcardTest.java

```diff
@@ -16,7 +16,7 @@ void givenFlashcardAttributes_whenInitialized_thenValuesAreSetCorrectly(){
     }
 
     @Test
-    void givenNewValues_whenSettersCalled_thenValuesAreUpdated() {
+    void givenNewValues_whenSettersCalled_thenValuesAreUpdated(){
         Flashcard card = new Flashcard("term1", "definition1", 0);
```

Comment:
Esta linea se repite en todos los test cases, considera usar un beforeEach en lugar de repetirlas con sus constantes 

```suggestion
    private static final String INITIAL_TERM = "term1";
    private static final String INITIAL_DEFINITION = "definition1";
    private static final String NEW_TERM = "newTerm";
    private static final String NEW_DEFINITION = "newDefinition";
    private Flashcard card;

    @BeforeEach
    void setUp() {
        card = new Flashcard(INITIAL_TERM, INITIAL_DEFINITION, 0);
    }
```